source/module_md/mdrun.h

```diff
@@ -0,0 +1,45 @@
+#ifndef MDRUN_H
+#define MDRUN_H
+
+#include "MD_parameters.h"
+#include "../module_cell/unitcell_pseudo.h"
+#include "../module_base/matrix.h"
+#include "module_esolver/esolver.h"
+
+class MDrun
+{
+public:
+    MDrun(MD_parameters& MD_para_in, UnitCell_pseudo &unit_in);
+    virtual ~MDrun();
+
```

Comment:
Please add annotations for functions in base class, which would be used for other developers in the future.